css/style.css

```
@@ -79,22 +92,21 @@ img {
 	margin: 0;
 	padding: 0;
 	display: flex;
+	justify-content: space-between;
 }
 
 .site_navigation li {
 	width: 240px;
 	text-align: center;
 	font-weight: bold;
 	color: var(--basic-dark);
-}
-
-.navigation_map, .navigation_hotel {
-
+	padding: 15px 15px;
```

```suggestion
	padding: 15px 0;
```
Зачем паддинги слева и справа?